src/components/Dropdown/Dropdown.tsx

```diff
@@ -59,7 +61,7 @@ export class Dropdown extends React.PureComponent<DropdownProps & React.HTMLAttr
     }
 
     public render() {
-        const { align, children, className, label, menuClass, ...attrs } = this.props;
+        const { align, children, className, label, menuClass, toggleDropdownOnClickInside, ...attrs } = this.props;
```

Comment:
Might consider being explicit and set `toggleDropdownOnClickInside = false` just so it is clearer on the default behavior.